projects/plugins/jetpack/_inc/client/at-a-glance/index.jsx

```diff
@@ -39,18 +39,6 @@ import { isOfflineMode, hasConnectedOwner } from 'state/connection';
 import { getModuleOverride } from 'state/modules';
 import { getScanStatus, isFetchingScanStatus } from 'state/scan';
 
-const renderPairs = layout =>
-	layout.map( ( item, layoutIndex ) => [
-		item.header,
-		item.pinnedBundle,
-		chunk( item.cards, 2 ).map( ( [ left, right ], cardIndex ) => (
-			<div className="jp-at-a-glance__item-grid" key={ `card-${ layoutIndex }-${ cardIndex }` }>
-				<div className="jp-at-a-glance__left">{ left }</div>
-				<div className="jp-at-a-glance__right">{ right }</div>
-			</div>
-		) ),
-	] );
-
```

Comment:
This looping was complex to understand and generating the `missing unique key prop` warning